packages/pwa-kit-dev/src/utils/resolver-utils.test.js

```diff
@@ -140,12 +146,34 @@ describe('resolverUtils', () => {
 
                 extensions: ['module-extension-a', 'module-extension-b', 'module-extension-c'],
```

Comment:
Should we also test edge cases to confirm we handled gracefully empty array or null extensions?
E.g. 
extensions: []
extensions: null